frontend/src/components/charger_frame.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Button } from './Button';
+
+function ChargerFrame() {
+  const textContent = [
+    // { label: 'Charger Name', value: 'Sample A' },
+    { label: 'State', value: 'Available' },
+    { label: 'Meter Value', value: '123456' },
+    { label: 'Max Power', value: '40A' },
+    { label: 'Current Power', value: '0' },
+  ];
+
+  return (
+    <div
+      style={{
+        width: '300px',
+        border: '2px solid #000',
+        borderRadius: '8px',
+        padding: '20px',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'flex-start',
+        alignItems: 'center',
+        backgroundColor: '#f9f9f9',
+      }}
+    >
+      <div
+        style={{
+          marginBottom: '20px', // Larger gap for Charger Name
+          fontSize: '18px', // Larger font for Charger Name
+          fontWeight: 'bold', // Bold Charger Name
+        }}
+      >
+        <p style={{ margin: '0', textAlign: 'left' }}>
+          <strong>Charger Name:</strong> Sample A
+        </p>
+      </div>
+
+      {/*the data displayed below for State, Meter value etc are temporary, will replace
+    with real data later*/}
+      {textContent.map((item, index) => (
+        <div key={index} style={{ marginBottom: '5px' }}>
+          {/* Smaller margin */}
+          <p style={{ margin: '0', textAlign: 'left' }}>
+            {/* Left-aligned text */}
+            <strong>{item.label}:</strong> {item.value}
+          </p>
+        </div>
+      ))}
+
+      <button
+        style={{
```

Comment:
The "Plug in Vehicle" button will result in an API call when we start implementing charging sessions. This button should be built with the `ButtonBase` class in `frontend/src/components/buttons/ButtonBase.js`.

Reply:
I think the bring online button is the outermost thing. It would be more reasonable that no button or charger status should be displayed in the offline state?

Reply:
That's a good point. If the online/offline toggle button is in an offline state, the other buttons should be hidden. Technically, the "Plug in Vehicle" action is possible when offline, but it will be complicated to use.

I think we can still display the text information. The charger won't pause charging if it is offline; it just cannot start a new charging session.

**Let's do this: if offline, hide all buttons that result in a message being sent to the backend.**